brokers_test/utils.go

```diff
@@ -0,0 +1,118 @@
+//
+// Copyright (c) 2012-2018 Red Hat, Inc.
+// This program and the accompanying materials are made
+// available under the terms of the Eclipse Public License 2.0
+// which is available at https://www.eclipse.org/legal/epl-2.0/
+//
+// SPDX-License-Identifier: EPL-2.0
+//
+// Contributors:
+//   Red Hat, Inc. - initial API and implementation
+//
+
+package brokers_test
```

Comment:
Why do you decide to choose package name with `_` ?
Please take a look the following article https://golang.org/doc/effective_go.html#package-names

I think it may be like `broker/common/test`
Also maybe it makes sense to move `mocks/Brokers` there if it is possible?

Reply:
mocks are autogenerated, I won't move them because of that. I'll move broker_test to /brokers/test